src/graph_notebook/notebooks/01-Getting-Started/01-About-the-Neptune-Notebook.ipynb

```diff
@@ -256,6 +259,107 @@
   {
    "cell_type": "markdown",
    "metadata": {},
+   "source": [
+    "## Issuing openCypher Queries\n",
+    "\n",
+    "You can issue an openCypher query using the `%%opencypher` (or just `%%oc`) cell magic. Let's populate the graph with a few elements, and then retrieve them."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "%%oc\n",
+    "\n",
+    "CREATE (n:Language {name:\"Gremlin\"})\n",
+    "CREATE (n:Language {name:\"Cypher\"})\n",
+    "CREATE (n:Language {name:\"SPARQL\"})"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "%%oc\n",
+    "\n",
+    "MATCH (n)\n",
+    "RETURN n\n",
+    "LIMIT 10"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Similar to the other query language magics, `%%oc` supports retrieval of a query explain plan."
```

Comment:
Do we need to change this? The query below is not for explain, but for bolt

Reply:
Removed this section.